spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -605,7 +606,8 @@ public static void importSparkPartitions(
           filesToImport
               .map((MapFunction<DataFile, String>) f -> f.path().toString(), Encoders.STRING())
               .toDF("file_path");
-      Dataset<Row> existingFiles = loadMetadataTable(spark, targetTable, MetadataTableType.ENTRIES);
+      Dataset<Row> entriesFiles = loadMetadataTable(spark, targetTable, MetadataTableType.ENTRIES);
```

Comment:
Style Nit: I think 'entriesFiles' doesnt make any sense.  

And also to reduce code, can we use something like 
```
loadMetadataTable(spark, targetTable, MetadataTableType.ENTRIES).filter("status != 2").
```

Seems there is the API, but not 100% sure it works as expected in Java.  That would be the best option to me.  Else we could maybe just call the df entries.

Reply:
thank you @szehon-ho , updated per your feedback